Docker/UsefulCommands.md

```diff
@@ -0,0 +1,141 @@
+# Useful Docker Commands
+Here follows a list of useful Docker commands with useful flags for each
+command.
+
+# Table of Contents
+0. [`docker build`](#docker-build)
+0. [`docker exec`](#docker-exec)
+0. [`docker images`](#docker-images)
+0. [`docker inspect`](#docker-inspect)
+0. [`docker logs`](#docker-logs)
+0. [`docker ps`](#docker-ps)
+0. [`docker rmi`](#docker-rmi)
+0. [`docker run`](#docker-run)
+0. [Learn More](#learn-more)
+
+
+## [`docker build`](https://docs.docker.com/engine/reference/commandline/build/)
+Build an image from a Dockerfile.
+
+```sh
+docker build [DOCKERFILE PATH]
+```
+
+### Example
+Build an image tagged `my-org:my-image` where the Dockerfile can be found at
```

Comment:
Shouldn't we use naming like the following: `my-org/my-image` ...?

Please do correct me if I am wrong.

Reply:
Yes, you are right. I mixed it up with the version `org/image:version`. Good catch!